### PR TITLE
chore(release): bump version to 1.0.1

### PR DESCRIPTION
**Pull request by pebie**
Release 1.0.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,24 @@
+## <small>1.0.1 (2021-09-16)</small>
```
Comment (Crow-EH) [inline CHANGELOG.md:1]:
```suggestion
## 1.0.1 (2021-09-16)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
+
+* 1.0.1 ([686b260](https://github.com/ekino/veggies/commit/686b260))
```
Comment (Crow-EH) [inline CHANGELOG.md:3]: wrong message / doesn't match the commit or its link.

Also, this isn't really useful IMO, we can omit it from the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (pebie) [reply CHANGELOG.md:1]: Fuck c'est `coventional-changelog` qui automatise ce truc. Je vais le virer à la main pour le moment.

File: CHANGELOG.md (hunk 2 above)
Comment (pebie) [reply CHANGELOG.md:3]: You're right it's probably a mistake during manual release process when I used the update changelog command.

I'will remove this line.

**Review by Crow-EH — APPROVED** [review]

**Comment by Crow-EH** [comment]
LGTM, you're free to merge it

**Pull request merged**